src/App.test.tsx

```diff
@@ -1,9 +1,14 @@
 import * as React from "react";
 import * as ReactDOM from "react-dom";
+import { act } from "react-dom/test-utils";
 import App from "App";
+import wait from "waait";
 
-it("renders without crashing", () => {
+it("renders without crashing", async () => {
   const div = document.createElement("div");
   ReactDOM.render(<App />, div);
+  await act(async () => {
+    await wait(0);
+  });
```

Comment:
what's this for?

Reply:
When testing, code that causes React state updates should be wrapped into act(...). In this case, we are waiting on the result of a promise that that causes changes to `GQLWrapper`.
https://github.com/evergreen-ci/spruce/blob/new-ui-feature-branch/src/utils/gql/GQLWrapper.tsx#L113